R/utils.R

```diff
@@ -2,11 +2,14 @@
 #' @param type Character string detailing type of widget to show
 get_filename = function(type) {
   filename = switch(type,
-                    "Logistic growth rate" = "tree-logistic_growth_rate-2021-11-27.html", # nolint
-                    "Simple logistic growth rate" = "tree-simple_logistic_growth_rate-2021-11-27.html", # nolint
-                    "Simple trait log odds" = "tree-sim_trait_logodds-2021-11-27.html" # nolint
+                    "Logistic growth rate" = "tree-logistic_growth_rate-2021-11-27.rds", # nolint
+                    "Simple logistic growth rate" = "tree-simple_logistic_growth_rate-2021-11-27.rds", # nolint
+                    "Simple trait log odds" = "tree-sim_trait_logodds-2021-11-27.rds" # nolint
```

Comment:
Do you want the dates embedded inside the file paths here? The app source code would need updating whenever the treeview() function is reran. The original code for making the .html files made a copy of the most-recent file that was named "tree-logistic_growth_rate.html". We could replicate that to make it simpler to always use the most-recent charts here

Reply:
I'm happy for this just to be "tree-logistic_growth_rate.html" since it will make it easier to update